docs/source/deployment/google_cloud_run.rst

```diff
@@ -67,7 +67,7 @@ This guide uses the IrisClassifier BentoService from the :doc:`Quick start guide
 
 .. code-block:: bash
 
-    > bentoml get IrisClassifier:latest
```

Comment:
why removing `>`, other bash commands all have this right?

Reply:
I think it is easier for user to copy.